### PR TITLE
changed: minor improvements to NL translation in Grammar, spelling, a…

### PR DESCRIPTION
**Pull request by Hobbesball**
…nd correct variable init.

Main changes:
- Capitalised words only if they were capitalised in English version
- removed space on line 20 so {%0} variable would be initialised correctly
- improved grammar and spelling (e.g. 'probeeren'->proberen, modern spelling e.g. 'Kwast naam'-> kwastnaam)

Could you please label this PR with the label 'hacktoberfest-approved'? That way we're sure this will count towards my hacktoberfest PR's ;) Thanks <3 

referencing #130

### CODE REVIEW

**Comment by inxomnyaa** [comment]
Oh, `hacktoberfest-approved`? looks like they've changed things, thy done

File: resources/lang/nld.ini [hunk 1]
```diff
@@ -17,13 +17,13 @@ error.limitexceeded = "U probeert te veel blokken tegelijk te bewerken. Verklein
 error.notarget = "Geen doelblok gevonden. Vergroot gereedschapsbereik met //setrange indien nodig"
 error.noselection = "Geen selectie gevonden - selecteer eerst een gebied"
 error.selectioninvalid = "De selectie is niet geldig! Controleer of alle posities zijn geselecteerd!"
-error.nosession = "Er is geen sessie gemaakt - waarschijnlijk geen toestemming om {% 0} te gebruiken"
+error.nosession = "Er is geen sessie gemaakt - waarschijnlijk geen toestemming om {%0} te gebruiken"
 error.noclipboard = "Geen klembord gevonden - maak eerst een klembord"
 warning.differentlevel = "[WAARSCHUWING] Je bewerkt een wereld waar je momenteel niet in zit!"
```
Comment (inxomnyaa) [inline resources/lang/nld.ini]: Oh btw this changed to .differentworld recently, mind patching?

**Review by inxomnyaa — CHANGES_REQUESTED** [review]

File: resources/lang/nld.ini (hunk 1 above)
Comment (Hobbesball) [reply resources/lang/nld.ini]: I've changed .nosession to .differentworld, is that what you intended?

**Comment by Hobbesball** [comment]
> Oh, `hacktoberfest-approved`? looks like they've changed things, thy done

hi, sorry I meant 'hacktoberfest-accepted', not approved, that's my bad! Could you change the label to hacktoberfest-accepted before merging?

File: resources/lang/nld.ini (hunk 1 above)
Comment (Aericio) [reply resources/lang/nld.ini]: should've been more specific, `differentlevel` -> `differentworld`

File: resources/lang/nld.ini [hunk 2]
```diff
@@ -17,13 +17,13 @@ error.limitexceeded = "U probeert te veel blokken tegelijk te bewerken. Verklein
 error.notarget = "Geen doelblok gevonden. Vergroot gereedschapsbereik met //setrange indien nodig"
 error.noselection = "Geen selectie gevonden - selecteer eerst een gebied"
 error.selectioninvalid = "De selectie is niet geldig! Controleer of alle posities zijn geselecteerd!"
-error.nosession = "Er is geen sessie gemaakt - waarschijnlijk geen toestemming om {% 0} te gebruiken"
+error.differentworld = "Er is geen sessie gemaakt - waarschijnlijk geen toestemming om {%0} te gebruiken"
 error.noclipboard = "Geen klembord gevonden - maak eerst een klembord"
 warning.differentlevel = "[WAARSCHUWING] Je bewerkt een wereld waar je momenteel niet in zit!"
```
Comment (Aericio) [inline resources/lang/nld.ini]:
```suggestion
warning.differentworld = "[WAARSCHUWING] Je bewerkt een wereld waar je momenteel niet in zit!"
```

File: resources/lang/nld.ini [hunk 3]
```diff
@@ -17,13 +17,13 @@ error.limitexceeded = "U probeert te veel blokken tegelijk te bewerken. Verklein
 error.notarget = "Geen doelblok gevonden. Vergroot gereedschapsbereik met //setrange indien nodig"
 error.noselection = "Geen selectie gevonden - selecteer eerst een gebied"
 error.selectioninvalid = "De selectie is niet geldig! Controleer of alle posities zijn geselecteerd!"
-error.nosession = "Er is geen sessie gemaakt - waarschijnlijk geen toestemming om {% 0} te gebruiken"
+error.differentworld = "Er is geen sessie gemaakt - waarschijnlijk geen toestemming om {%0} te gebruiken"
```
Comment (Aericio) [inline resources/lang/nld.ini]:
```suggestion
error.nosession = "Er is geen sessie gemaakt - waarschijnlijk geen toestemming om {%0} te gebruiken"
```

File: resources/lang/nld.ini [hunk 4]
```diff
@@ -75,14 +75,14 @@ tool.wand.lore.1 = "Klik met de linkermuisknop op een blok om de positie 1 van e
 tool.wand.lore.2 = "Klik met de rechtermuisknop op een blok om de positie 2 van een selectie in te stellen"
 tool.wand.lore.3 = "Gebruik //togglewand om van functionaliteit te wisselen"
 tool.wand.disabled = "De toverstaf is uitgeschakeld. Gebruik //togglewand om het opnieuw in te schakelen"
-tool.wand.setenabled = "Het gereedschap Toverstaf is nu{ %0}!"
+tool.wand.setenabled = "De toverstaf is nu{ %0}!"
```
Comment (Aericio) [inline resources/lang/nld.ini]:
```suggestion
tool.wand.setenabled = "De toverstaf is nu {%0}!"
```

**Review by Aericio — CHANGES_REQUESTED** [review]

**Comment by Aericio** [comment]
> Oh, hacktoberfest-approved? looks like they've changed things, thy done

Maybe it is because of the `Amazing Project` PR spam that has been happening.
https://github.com/search?q=Amazing+Project&type=issues

---

edit: Looks like that's the case, https://hacktoberfest.digitalocean.com/hacktoberfest-update
> After working closely with our friends at GitHub, we are happy to introduce a new measure to help significantly reduce the amount of spammy contribution. We’re making Hacktoberfest opt-in only for projects – which maintainers can do simply by adding the ‘hacktoberfest’ topic to a repository.

> Updated (10/05/2020 15:00 UTC): Individual pull requests can also be directly opted-in to Hacktoberfest with the ‘hacktoberfest-accepted’ label, on any public GitHub repository, without the need for the ‘hacktoberfest’ repository topic.

**Comment by inxomnyaa** [comment]
.nosession and .differentworld, something looks like it got lost

**Review by Aericio — APPROVED** [review]

**Pull request merged**